fluxion/decoders/gstreamer.py

```diff
@@ -47,10 +47,10 @@ def decode(self, input_filepath: str, output_filepath: str):
         pipeline = PIPELINE_TPL.format(self.cmd, input_filepath,
                                        self.decoder_bin, self.caps, output_filepath)
         subprocess.run(shlex.split(pipeline),
-                       stdout=subprocess.DEVNULL, check=True)
+                       stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, check=True, timeout=5)
```

Comment:
I would like to see `timeout` as a parameter :)

Reply:
Done